.gitignore

```diff
@@ -19,3 +19,4 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 package-lock.json
+yarn.lock
```

Comment:
Please do not edit the .gitignore. Can you remove this from the commit?

Reply:
Will add another commit to delete that line from .gitignore